lib/event/domain/register/success.php

```diff
@@ -62,50 +67,6 @@ public function get_domain_status(): Entity\Epp_Status_Codes {
 		return new Entity\Epp_Status_Codes( ...$epp_statuses );
 	}
 
-	/**
-	 * Gets the date the domain was created
-	 *
-	 * @return null|\DateTimeInterface
-	 */
-	public function get_created_date(): ?\DateTimeInterface {
-		$created_date = $this->get_data_by_key( 'event_data.created_date' );
-
-		return null === $created_date ? null : Helper\Date_Time::createImmutable( $created_date );
-	}
-
-	/**
-	 * Gets the domain expiration date
-	 *
-	 * @return null|\DateTimeInterface
-	 */
-	public function get_expiration_date(): ?\DateTimeInterface {
-		$expiration_date = $this->get_data_by_key( 'event_data.expiration_date' );
-
-		return null === $expiration_date ? null : Helper\Date_Time::createImmutable( $expiration_date );
-	}
-
-	/**
-	 * Get the last date to renew the domain
-	 *
-	 * @return null|\DateTimeInterface
-	 */
-	public function get_renewable_until(): ?\DateTimeInterface {
-		$renewable_until = $this->get_data_by_key( 'event_data.renewable_until' );
-
-		return null === $renewable_until ? null : Helper\Date_Time::createImmutable( $renewable_until );
-	}
-
-	/**
-	 * Gets the date when the domain will be suspended if the contact information is not verified
-	 *
-	 * @return null|\DateTimeInterface
-	 */
-	public function get_unverified_contact_suspension_date(): ?\DateTimeInterface {
-		$unverified_contact_suspension_date = $this->get_data_by_key( 'event_data.unverified_contact_suspension_date' );
-
-		return null === $unverified_contact_suspension_date ? null : Helper\Date_Time::createImmutable( $unverified_contact_suspension_date );
-	}
-
```

Comment:
Should we remove those methods from the `Domain/Register/Success` event? These data are still returned from the `Domain/Register` command, aren't they?

Reply:
@leonardost I removed those methods since I moved the to treats :)